core/Http.php

```diff
@@ -22,7 +22,7 @@ class Http
     /**
      * Returns the "best" available transport method for {@link sendHttpRequest()} calls.
      *
-     * @return string Either `'curl'`, `'fopen'` or `'socket'`.
+     * @return string|null Either curl, fopen, socket or null of no method is supported.
```

Comment:
FYI, not sure if you know this, but it should get outputted as inline code on developer.piwik.org if backticks are used (since it's markdown). If it's more readable, though there's no issue.